### DISCUSSION TITLE
Running CHGNet on multiple GPU's

### DISCUSSION THREAD

**Discussion by dundaryilmaz** [CederGroupHub/chgnet#64] [General]
Hi, 
Is it possible to run CHGNet on multiple GPUs? If possible could you refer an example?
Or, to simulate a large system where a RT3080 with 12gb ram gives a memory error is there a work around?

**Comment by bowen-bd** [comment]
Hi,

CHGNet parallel GPU inference currently can be supported through LAMMPS.
For the LAMMPS interface, please refer to: 
https://github.com/CederGroupHub/chgnet/issues/57
and
https://docs.lammps.org/Developer_parallel.html

We also recomment you to consider using larger memory GPU or decrease your system size if possible
